packages/toolpad-core/src/createQuery.tsx

```diff
@@ -23,21 +31,37 @@ export interface CreateQueryConfig<P> {
 type CreateQueryConfigParameters<C extends CreateQueryConfig<CreateQueryConfigParameters<C>>> =
   QueryResolverParams<C>['parameters'];
 
+/**
+ * @deprecated
+ * QueryResolverParams is deprecated. Use FunctionResolverParams instead.
+ */
 export interface QueryResolverParams<C extends CreateQueryConfig<CreateQueryConfigParameters<C>>> {
   parameters: {
     [K in keyof C['parameters']]: QueryParameterTypeLookup[C['parameters'][K]['typeDef']['type']];
   };
 }
 
+/**
+ * @deprecated
+ * QueryResolver is deprecated. Use FunctionResolver instead.
+ */
 export interface QueryResolver<C extends CreateQueryConfig<CreateQueryConfigParameters<C>>> {
   (params: QueryResolverParams<C>): Promise<unknown>;
 }
 
+/**
+ * @deprecated
+ * ToolpadQuery is deprecated. Use ToolpaFunction instead.
+ */
 export interface ToolpadQuery<C extends CreateQueryConfig<CreateQueryConfigParameters<C>>>
   extends QueryResolver<C> {
   [TOOLPAD_QUERY]: C;
 }
 
+/**
+ * @deprecated
+ * createQuery is deprecated. Use createFunction instead.
+ */
 export default function createQuery<C extends CreateQueryConfig<CreateQueryConfigParameters<C>>>(
```

Comment:
Is it possible to avoid duplication this way?

```tsx
import createFunction from './createFuction'

/**
 * @deprecated
 * createQuery is deprecated. Use createFunction instead.
 */
const createQuery = createFunction
export default createQuery
```